evaluator/opa.go

```diff
@@ -39,6 +39,19 @@ type OpaEvaluator struct {
 	bundleServer *http.Server
 }
 
+var opaConfig = `{
```

Comment:
It would be nice if we will able to enable the decision logs while debugging with a flag or something

Reply:
We don't have the infra for that at the moment so I think it's out of scope.